BasicVariables.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HelloWorld
+{
+    class BasicVariables
+    {
+        int intVariable;
+        long longVariable;
+        double doubleVariable;
+        float floatVariable;
+        string stringVariable;
+
+        /// <summary>
+        /// Compare unassigned values for int, long, float and double
+        /// </summary>
+        public void CompareUnassignedValues()
+        {
+            Console.WriteLine($"Compared Int and Long: {intVariable==longVariable}");
+            Console.WriteLine($"Compared Long and Double: {longVariable == doubleVariable}");
+            Console.WriteLine($"Compared Float and Int: {floatVariable == intVariable}");
+        }
+
+        /// <summary>
+        /// Compare different values for int, long, float and double
+        /// </summary>
+        public void CompareDifferentAssignedValues()
+        {
+            intVariable = 50;
+            longVariable = 1234567890123;
+            doubleVariable = 123.45678;
+            floatVariable = 456.45F;
+            stringVariable = "String Variable";
```

Comment:
try smth like string stringVariable = "5"; 
and then compare via: stringVariable.Equals(intVariable)